tests/c_languages/codeclone_detection/ClangASTPrintBearTest.py

```diff
@@ -52,6 +52,6 @@ def test_ast(self):
         ast = "\n"
         # Only check beginning of AST
         for i in range(expected_ast.count("\n")-1):
-            ast += self.queue.get(timeout=0).message + "\n"
+            ast += self.queue.get(timeout=0.1).message + "\n"
```

Comment:
Can you elaborate? IMO *No whitespace at all* seems misguiding here.
Clearly, he needs to add a whitespace around `+` and no whitespace around
function argument. Is there anything else you are pointing to?
ᐧ

On Thu, Mar 31, 2016 at 5:57 PM, Sanket Dasgupta <notifications@github.com>
wrote:

> In tests/c_languages/codeclone_detection/ClangASTPrintBearTest.py
> <https://github.com/coala-analyzer/coala-bears/pull/308#discussion_r58044524>
> :
>
> > @@ -52,6 +52,6 @@ def test_ast(self):
> >          ast = "\n"
> >          # Only check beginning of AST
> >          for i in range(expected_ast.count("\n")-1):
> > -            ast += self.queue.get(timeout=0).message + "\n"
> > +            ast += self.queue.get(timeout=0.1).message + "\n"
>
> actually there should be no whitespace at all
> http://pep8.org/#other-recommendations
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/308/files/0af67c2971fd237e1f06dedbb7566982c11ba8cd#r58044524>
>